[Insights] Update Custom Reports docs (#551)
wso2/docs-choreo-dev: en/docs/insights/generate-custom-reports.md

Choreo insights allow you to generate custom reports to view the information that is important to you to make timely decisions for the betterment of your business. With custom reports, you can generate reports for a set of metrics of your choice and aggregate results by specific fields. Once generated, the metrics can be filtered out using any selected group-by filters. Custom reports support three visualizations: overtime charts, pie charts, and tables.

## Metrics

> Choreo Insights allow you to retrieve data for the following metrics,

Reviewer: ```suggestion
Choreo Insights allow you to retrieve data for the following metrics:
```

 - Successful Hit Count
 - Response Cache Hits
 - Request Mediation Latency
 - Response Mediation Latency
 - Backend Latency
 - Total Latency
 - API Errors
 - Target Errors

> ## Group-by Fields

Reviewer: ```suggestion
## Group-by fields
```

> Group-by Fields determine how the metric data should be grouped. As an example you could retrieve Successful Hit Count **grouped by API Name**

Reviewer: ``` suggestion
**Group-by** fields specify how the metric data should be grouped. e.g., You can select **Successful Hit Count** in the **Metrics** field and **API Name** in the **Group-by** fields to retrieve the successful hit count grouped by the API name.
```

> Choreo Insights allow you to group metric data by the following fields,

Reviewer: ```
Choreo Insights allow you to group metric data by the following fields:
```

 - API Name
 - API Version
 - API Resource Template
 - API Method
 - API Creator
 - Application
 - Application Owner
 - Destination
 - User Agent
 - Platform

To generate a custom report, follow the steps below:

> 1. Click on **Custom Reports** from the left navigation 

Reviewer: ```
1. In the left navigator, click **Custom Reports**.
```

6. Click **Generate**.

    ![Generate](../assets/img/insights/custom-reports-step-6.png){.cInlineImage-full}

## Download Reports

> 

Reviewer: Since there is only one step, shall we have it in paragraph form?

> Choreo Insights allow you to download report data for each chart as either **PDF** or **CSV** files. To download report data, continue on with the following steps, 

Reviewer: ```suggestion
Choreo Insights allow you to download report data related to each chart as a PDF or a CSS file. 

To do this, click the **Download** icon on the top right of the relevant chart and select the required file format.
```
